<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanFrame</class>
 <widget class="QWidget" name="PlanFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#scrollArea{border:none;}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#scrollAreaWidgetContents{background:#404244;}</string>
    </property>
   </widget>
  </widget>
  <widget class="FButton" name="sortBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/image/dark/resource/image/dark/sort.png</normaloff>:/image/dark/resource/image/dark/sort.png</iconset>
   </property>
   <property name="borderRadius">
    <number>25</number>
   </property>
   <property name="bgColor">
    <string>#303133</string>
   </property>
   <property name="maxShadow">
    <number>15</number>
   </property>
  </widget>
  <widget class="FButton" name="fButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>28</pointsize>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string>添加新计划</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/image/dark/resource/image/dark/add_file_2.png</normaloff>:/image/dark/resource/image/dark/add_file_2.png</iconset>
   </property>
   <property name="borderRadius">
    <number>25</number>
   </property>
   <property name="bgColor">
    <string>#303133</string>
   </property>
   <property name="maxShadow">
    <number>15</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FButton</class>
   <extends>QPushButton</extends>
   <header>fbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
